### DISCUSSION TITLE
SONOFF Zigbee 3.0 USB Dongle Plus installation problem

### DISCUSSION THREAD

**Discussion by SmartKobol** [Koenkk/zigbee2mqtt#13171] [Devices]
Dear Alls, 
Trivial problem, but rather annoying by now. Please help! Having a SONOFF Zigbee 3.0 USB Dongle Plus with original firmware, no flash. HA detects it (#1) Port is adding as suggested in the "Adapter settings" (#2) Logs says error in serial/adapter (#3) WebUI does not start, of course: "502: Bad gateway".  Nevertheless ZHA detects the dongle and works with without any problem. Where did I go wrong? 
<img width="407" alt="#1" src="https://user-images.githubusercontent.com/81106324/179371330-f0328f83-102e-471f-a32e-71b511530a02.PNG">
<img width="718" alt="#2" src="https://user-images.githubusercontent.com/81106324/179371332-c100a939-d419-4e8c-ba10-84ce23a2a320.PNG">
<img width="530" alt="#3" src="https://user-images.githubusercontent.com/81106324/179371333-85347dbe-f02d-4f33-850c-32ec64b59ddd.PNG">

**Comment by Koenkk** [comment]
Try removing the `>-` after the `port:` (so everything is on 1 line)

**Reply by SmartKobol** [reply]
Well I have tried. I delete the >-, all in 1 line, then save, restart add-on and boom, if I open the 'configuration' it appears again and again... Nevertheless it is weird that in the configuration.yaml ok (#4). What else can I do?
<img width="311" alt="#4" src="https://user-images.githubusercontent.com/81106324/179425934-ea372b9c-3e6c-40dd-a607-d6d626a8a760.PNG">

**Reply by Koenkk** [reply]
Under the `port:` can you add: `adapter: zstack`?
